src/libs/antares/study/area/list.cpp

```diff
@@ -937,12 +937,15 @@ static bool AreaListLoadFromFolderSingleArea(Study& study,
     }
 
     // Short term storage
+    if (study.header.version >= 860)
```

Comment:
We load input data for ST storage clusters at 2 different places : 
- we create the cluster in one place
- we load the time series in another place

Why is that ?
Maybe we want to do as we do for loading **thermal clusters** for example.
But I'm not sure it is done properly regarding **thermal clusters**.

We should load ST storage clusters and time series in one place, for clarity.

Reply:
Eventually this problem was solved.
@flomnes : thx for your precious technical help !

Reply:
We could but this will indeed break consistency with other clusters like thermal, where series and prepro are loaded in a different function